src/App.js

```diff
@@ -2,15 +2,61 @@ import React from 'react';
 import './App.scss';
 
 class App extends React.Component {
-  addOne = () => {};
+  state = {
+    count: 0,
+  }
+
+  addOne = () => {
+    this.setState(prevState => ({
+      count: prevState.count + 1,
+    }));
+  };
 
-  add100 = () => {};
+  add100 = () => {
+    this.setState(prevState => ({
+      count: prevState.count + 100,
+    }));
+  };
 
-  increase = () => {};
+  increase = () => {
+    this.setState(prevState => (
+      prevState.count % 5 === 0 ? this.add100() + this.addOne() : this.addOne()
+    ));
+  };
```

Comment:
не пойму что тут происходит
```suggestion
  increase = () => {
    const { count } = this.state;
    
    if (count % 5 === 0) {
      // code
    } else {
      // code
    }
  };
```